kubernetes/oauth2-proxy.libsonnet

```diff
@@ -24,7 +24,7 @@ local ok = import 'kubernetes/kube.libsonnet';
     // - OAUTH2_PROXY_COOKIE_SECRET: https://oauth2-proxy.github.io/oauth2-proxy/docs/configuration/overview#generating-a-cookie-secret
     // - OAUTH2_PROXY_CLIENT_ID: Client ID of your IT provided Okta ODIC app
     // - OAUTH2_PROXY_CLIENT_SECRET: Client Secret of your IT provided Okta ODIC app
-    secret=error 'secret must be set',
+    secretName=error 'secret must be set',
```

Comment:
This isn't backwards-compatible. Did you confirm no one is using it the other way ... ?

Reply:
I also prefer passing in the secret object. I'd suggest having an optional `secretName` param that can be used instead.

Reply:
I did not find any usage of this library prior to opening this PR. I'll concede and have it ben an optional parameter, but I think this is both safe and makes the library more accessible since we currently do not declare secret objects that can quickly be passed in.